app/src/main/java/com/heig/atmanager/User.java

```diff
@@ -324,6 +329,73 @@ private  Date getDateXDaysAgo(int numberOfDaysAgo){
         return todos;
     }
 
+    public ArrayList<GoalTodo> getDailyGoalTodo(){
+        ArrayList<GoalTodo> result = new ArrayList<>();
+        Calendar current = Calendar.getInstance();
+        Date currentDate = new Date();
+        current.setTime(currentDate);
```

Comment:
Is the Date really necessary ? Calendar.getInstance is already set to the current time no ?